Libraries/Hotcakes.PaypalWebServices/Properties/AssemblyInfo.cs

```diff
@@ -9,4 +9,3 @@
 // The following GUID is for the ID of the typelib if this project is exposed to COM
 [assembly: Guid("2bc855dd-49b7-4d2d-96f9-ef76dd05e945")]
 
-[assembly: AllowPartiallyTrustedCallers()]
```

Comment:
Why is this removed? Does it fix anything?  What are the pros and cons of keeping/removing it?

Reply:
When I tried to use any function of this class from another problem it doesn't allow it and in the forums the suggested solution was to remove that line.